package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "backend",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "dependencies": {
+    "dotenv": "^8.2.0",
+    "express": "^4.17.1",
+    "nodemailer": "^6.4.6",
+    "nodemailer-sendgrid-transport": "^0.2.0",
+    "pg": "^8.0.2",
+    "sequelize": "^5.21.6"
+  },
+  "devDependencies": {
+    "@types/bluebird": "^3.5.30",
+    "@types/express": "^4.17.6",
+    "@types/node": "^13.11.1",
+    "@types/validator": "^13.0.0",
+    "sequelize-cli": "^5.5.1",
+    "typescript": "^3.8.3"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "npm run build && node ts-built/index.js",
```

Comment:
I'm assuming this currently requires a manual restart, and we'll add nodemon in later?

Reply:
Yup

Reply:
I didn't have a chance to get nodemon, with typescript working, especially getting it to work with stack traces (source maps) currently we do this with ts-node